unreleased_history/performance_improvements/parallel_notify-one_i_LaunchParallelMemTableWriters.md

```diff
@@ -0,0 +1,18 @@
+# Improved write throughput to memtable when there's a large number of concurrent writers and allow_concurrent_memtable_write=true(#12545)
+
+To wake up each writer to write its own memtable, the leader writer first wakes up the (n^0.5-1) caller writers, and then those callers and the leader will wake up n/x separately to write to the memtable. This reduces the number for the leader's to SetState n-1 writers to 2*(n^0.5) writers in turn.
+
+---
+
+vcpu=160, benchmark=db_bench
+The score is normalized:
+| case name         | optimized/base |
+|-------------------|----------------|
+| fillrandom        | 182%           |
+| fillseq           | 184%           |
+| fillsync          | 136%           |
+| overwrite         | 179%           |
+| randomreplacekeys | 180%           |
+| randomtransaction | 161%           |
+| updaterandom      | 163%           |
+| xorupdaterandom   | 165%           |
```

Comment:
I'd suggest removing this part since they are available in the PR summary. Otherwise LGTM.

Reply:
Thank you, I have removes this part.